benchmarking/src/performance_evaluation.py

```diff
@@ -130,7 +131,11 @@ def adjust_to_exact_tokens(self, text: str, target_token_count: int) -> str:
         """
         model = self.model_name.replace('Bundle/', '').replace('/', '_')
         tokenized_text_filename = f'tokenized_text_variable_{model}.bin'
-        tokenized_text_filepath = f'{file_location}/../prompts/{tokenized_text_filename}'
+        tokenized_text_directory = f'{kit_location}/../../persistent_files/prompts'
```

Comment:
It should be

`f'{kit_location}/../../scratch/benchmarking/persistent_files/prompts'`

@snova-kwasia Could you also confirm?

Reply:
@snova-francescar - Correct.   @snova-rodrigom can you double check these paths? Is there a root scratch location you're referring to.